CFC_WebApp/main/K_medoid_2.py

```diff
@@ -15,9 +15,10 @@
 def user_route_data(user_id, database):
     data_feature = {}
 
-    for section in database.find({'$and':[{'user_id': user_id},{'type': 'move'},{'confirmed_mode': {'$ne': ''}}]}):
+    # for section in database.find({'$and':[{'user_id': user_id},{'type': 'move'},{'confirmed_mode': {'$ne': ''}}]}):
+    for section in database.find({'$and':[{'user_id': user_id},{'type': 'move'}]}):
```

Comment:
Long-term, should probably generalize to a query instead so that we can potentially support client specific queries.